pkcs7/pkcs7-verify.c

```diff
@@ -0,0 +1,69 @@
+#include <wolfssl/options.h>
+#include <wolfssl/wolfcrypt/settings.h>
+#include <wolfssl/wolfcrypt/pkcs7.h>
+#include <wolfssl/wolfcrypt/error-crypt.h>
+#include <wolfssl/wolfcrypt/logging.h>
+
+#ifdef HAVE_PKCS7
+
+int main(int argc, char** argv)
+{
+    int rc = 0;
+    PKCS7 pkcs7;
+    XFILE derFile;
+    byte* derBuf = NULL;
+    word32 derSz = 0;
+
+#ifdef DEBUG_WOLFSSL
+    wolfSSL_Debugging_ON();
+#endif
+
+    /* load DER PKCS7 */
+    derFile = fopen("signed.p7s", "rb");
+    if (derFile) {
+        fseek(derFile, 0, SEEK_END);
+        derSz = (int)ftell(derFile);
```

Comment:
derSz is a word32 but type-casting result of ftell to int... seems like cast should be to word32 instead!